mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -98,6 +98,8 @@ CLASS ahm net/minecraft/entity/Entity
 	METHOD a setRotation (FF)V
 		ARG 1 yaw
 	METHOD a updateVelocity (FLcnl;)V
+		ARG 1 movementSpeed
```

Comment:
Can you just name it `speed`? Same for below.

Reply:
Could do, but it's [sufficiently complicated](https://github.com/FabricMC/yarn/pull/485#issuecomment-465793034) that I don't think 'speed' is descriptive enough. The method is not as simple as `direction * speed` (i.e. the length of the calculated velocity vector is not guaranteed to `== speed`). `movementSpeed` is much more accurate to what's being passed to the function, as it's more like a base movement speed than a speed that is directly applied.

To give a real example, the movementSpeed parameter is the same when both walking and crouching, but the movementInput components are 0.3 of normal when crouching.

Reply:
How about `attributeSpeed` as this is sound like calculated with vanilla entity attributes?

Reply:
Callers do many different transformations to the value they pass to `updateVelocity`. Very rarely is it an unmodified `EntityAttributes.MOVEMENT_SPEED` (only instance I see of that is in `DolphinEntity.travel`, and even then it's actually `getMovementSpeed()` which the field it gets is only sometimes set to `EntityAttributes.MOVEMENT_SPEED`). For example, in the [code snippet in my comment here](https://github.com/FabricMC/yarn/pull/485#issuecomment-465793034), friction gets applied to the value before passing it to the function. And a lot of the time it's called with constant values, i.e. here's a call inside FlyingEntity.travel:

```
if (this.isInsideWater()) {
    this.updateVelocity(0.02f, movementInput);
```